src/vtquery.cpp

```diff
@@ -214,7 +208,9 @@ struct Worker : Nan::AsyncWorker {
                         // get the geometry type and decode the geometry into mapbox::geometry data structures
                         switch (feature.geometry_type()) {
                         case vtzero::GeomType::POINT: {
-                            if (data.geometry.size() > 0 && data.geometry != "point") continue;
+                            if (!data.geometry.empty() && data.geometry != "point") {
```

Comment:
Rather then using a string comparison here, it would be better to compare the value to an enum. This would be much faster to compare.

Reply:
Enums added in ecf50caf76696a7b2e699f2d49f2b0c8ad3a7385 and 82fe175dd9e4c950f29677ae14839d2750d9b2fb @flippmoke